2019/04/2019-04-23-bilibili.md

```diff
@@ -1,18 +1,16 @@
 Hello Dear Github:
 
-  We Are:
-  
-  Bilibili Security Group.It is nice to be able to yse Github such a professional and effcient platform,and mpw we have a need to be very hopefully able to get Github's support and resolution.I sent the mail yesterday and the mail attachments is the my signature!!
+We Are Bilibili Security Group. It is nice to be able to use Github such a professional and effcient platform, and now we have a need and  hopefully we are able to get Github's support and resolution. I sent the mail yesterday and the mail attachment is my signature!!
 
-We have read and understand GitHub's Guide to Filling a DMCA Notice
+We have read and understand GitHub's Guide to Filling a DMCA Notice.
```

Comment:
understand -> understood.
to filling -> to fill.

Reply:
It's actually to file